src/roman-numerals/index.js

```diff
@@ -4,6 +4,35 @@
  * @param {string} roman The all-caps Roman numeral between 1 and 3999 (inclusive).
  * @returns {number} The decimal equivalent.
  */
-function romanToDecimal(roman) {}
+function romanToDecimal(roman) {
+  let final = 0;
+  let cur;
+  let curVal;
+  let next;
+  let nextVal;
+  const newObject = new Object();
```

Comment:
Why not just use a regular object here.

Reply:
Okay.. will use a regular Object next time.
Thank you.